chip-testing/test/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "../../tools/tsconfig.base.json",
+    "compilerOptions": {
+        "outDir": "../build/test",
+        "target": "es2018",
+        "module": "nodenext",
```

Comment:
Am I correct in understanding module & moduleResolution are just here because you moved them from a level up?  Probably we should just remove and use the default node16 but doesn't have to be this PR.

Reply:
Yes, I just kept them from the original upper level `tsconfig.json`.